docs/GUIDES.md

```diff
@@ -177,8 +177,242 @@ Our playing field is a grid of tiles, addressable like a coordinate system:
    │ 0│ 1│ 2│ 3│ 4│ 5│ 6│ 7│ 8│ 9│10│11│12│13│14│15│16│17│18│19│20│
 ```
 
+- the whole game operates in a predefined grid:
+  - snake an food for the snake will be drawn as filled boxes in the grid
+  - movement only along the predefined grid
+- in Scenic terms, our game will be a "scene"
+  - see https://hexdocs.pm/scenic/overview_general.html#architecture
+  - terms and definitions for Scenic: https://hexdocs.pm/scenic/overview_general.html#terms-and-definitions
+  - scenes are like web pages:
+    - every scene is a `GenServer` process containing state and the business logic to handle user input
+    - navigating to different screens means navigate to different "scenes"
+  - for our game, one scene is enough for the start as we won't navigate anywhere else
```

Comment:
There is also the [GameOver scene](https://github.com/codecurious-bln/elixir_snake/blob/f6d3ec01c1e04302f724b31a252e96ad0c26776d/lib/scenes/game_over.ex). Maybe something like "the main game will consist of only one scene"?